AmazonChimeSDK/AmazonChimeSDK/ingestion/IngestionEventConverter.swift

```diff
@@ -67,13 +68,13 @@ import Foundation
             return nil
         }
 
-        let rootMeta = toIngestionMetadata(attributes: EventAttributeUtils.getCommonAttributes())
+        let rootMeta = toIngestionMetadata(attributes: EventAttributeUtils.getCommonAttributes(ingestionConfiguration: ingestionConfiguration))
```

Comment:
Is adding the ingestionConfiguration param related to this event metrics? Have not seen this is used in this PR.

Reply:
getCommonAttributes(ingestionConfiguration: ingestionConfiguration) will add meetingId and attendeeId to metadata, which is required by JS team.